packages/client/webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: process.env.NODE_ENV,
+  entry: path.join(__dirname, "./src/index.tsx"),
+  output: {
+    path: path.resolve(__dirname, "../server/public/dist"),
+    publicPath: "/dist/",
+    filename: "bundle.js",
+  },
+  devServer: {
```

Comment:
This is needed so we can serve the React application in hot-reload mode, but still have it in live communication with a server process.